src/main/java/com/kuit/agarang/domain/login/controller/ReissueController.java

```diff
@@ -0,0 +1,106 @@
+package com.kuit.agarang.domain.login.controller;
+
+import com.kuit.agarang.domain.login.jwt.JWTUtil;
+import com.kuit.agarang.domain.member.model.entity.RefreshToken;
+import com.kuit.agarang.domain.member.repository.RefreshRepository;
+import io.jsonwebtoken.ExpiredJwtException;
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Date;
+
+@RestController
+@RequiredArgsConstructor
+public class ReissueController {
+
+  private final JWTUtil jwtUtil;
+  private final RefreshRepository refreshRepository;
+  @Value("${secret.jwt-access-expired-in}")
+  private Long ACCESS_EXPIRED_IN;
+  @Value("${secret.jwt-refresh-expired-in}")
+  private Long REFRESH_EXPIRED_IN;
+
+  @PostMapping("/reissue")
+  public ResponseEntity<?> reissue(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
```suggestion
  public ResponseEntity<BaseResponse<ResponseDto>> reissue(HttpServletRequest request, HttpServletResponse response) {
```

와일드 카드로 처리하기 보다는 확실히 응답형태를 지정해주면 좋을 것 같아요!

Reply:
확인했습니다!!